src/etc/inc/radius.inc

```diff
@@ -696,6 +696,26 @@ class Auth_RADIUS extends PEAR {
                     }
                 }
 
+                elseif ($vendor == 13644) { /* Netgate */
+                    switch ($attrv) {
+                    case 1: /* pfSense-Max-Input-Octets */
+                        $this->attributes['maxbytesup'] = radius_cvt_int($datav);
+                        break;
+                    case 2: /* pfSense-Max-Output-Octets */
+                        $this->attributes['maxbytedown'] = radius_cvt_int($datav);
```

Comment:
On second thought it would probably be better just not to add pfSense-Max-{Input,Output}-Octets if they're not used and do nothing. Somebody could try to use them.